Classes/Controllers/ApplicationController.m

```diff
@@ -143,10 +143,10 @@ - (IBAction)showAboutPanel:(id)sender
 		[dict addEntriesFromDictionary:[[NSDictionary alloc] initWithObjectsAndKeys:gitversion, @"Version", nil]];
 
 	#ifdef DEBUG_BUILD
-		[dict addEntriesFromDictionary:[[NSDictionary alloc] initWithObjectsAndKeys:@"GitX-dev (DEBUG)", @"ApplicationName", nil]];
+		[dict addEntriesFromDictionary:[[NSDictionary alloc] initWithObjectsAndKeys:@"GitX(N) (DEBUG)", @"ApplicationName", nil]];
```

Comment:
Why not just go with GitX and no qualifier? Is there a system I'm unaware of?

GitX(NG) (next-gen) is another option?

Or... GitX(2021) or GitX(202X)? That'll make discovery way easier.